python/ray/llm/BUILD

```diff
@@ -0,0 +1,19 @@
+load("//bazel:python.bzl", "doctest")
+
+doctest(
+    files = glob(["**/*.py"], exclude=glob(["examples/**/*", "tests/**/*"])),
+    tags = ["team:llm"]
+)
+# --------------------------------------------------------------------
+# Tests from the python/ray/llm/tests directory.
+# Covers all tests starting with `test_`.
+# Please keep these sorted alphabetically.
+# --------------------------------------------------------------------
+load("//bazel:python.bzl", "py_test_module_list")
+
+py_test_module_list(
+    files = glob(["tests/batch/**/test_*.py"]),
```

Comment:
you want to do `tests/**/test_**.py` or you want to separate batch from non batch tests?

Reply:
Removed this file from this PR as it will be added by #50041